spec/presenters/period_contribution_calculator_presenter_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe Wpcc::PeriodContributionCalculatorPresenter do
+  subject do
+    described_class.new(pcc, view_context: view_context)
+  end
+
+  let(:pcc) { Wpcc::PeriodContribution.new }
```

Comment:
😱  `pcc` does this stand for `Pensions Contribution Calculator` like in `wpcc`? 
I'm guessing this has been abrreviated to avoid Rubocop line lengths, but I don't know if this is the correct solution to deal with it...